setup.py

```diff
@@ -5,15 +5,16 @@
 with open('requirements.txt') as f:
     requirements = f.read().split()
 
-# Temporary hack until databroker is on PyPI, needed because
-# `pip install -r requirements.txt` works with git URLs, but `install_requires`
-# does not.
-requirements = [r for r in requirements if not r.startswith('git+')]
-print("User must install https://github.com/NSLS-II/databroker manually")
 
 setup(name='ophyd',
       version=versioneer.get_version(),
       cmdclass=versioneer.get_cmdclass(),
       license='BSD',
       install_requires=requirements,
-      packages=find_packages())
+      packages=find_packages(),
+      classifiers=[
+        "Development Status :: 5 - Production/Stable",
+        "Programming Language :: Python :: 3.5",
+        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
+      ])
```

Comment:
@tacaswell, since you already touched `setup.py`, do you mind to also update it to add proper description to the PyPI page? Please see https://github.com/NSLS-II/databroker/pull/369 for more details. The changes would be:
```py
# Get the long description from the README file
with open(path.join(here, 'README.md'), encoding='utf-8') as f:
long_description = f.read()

setup(
    ...
    long_description=long_description,
    long_description_content_type='text/markdown',
    ...
)
```

Reply:
Looks good to me now! Thanks @klauer for the reminder :)